wsgi/odyssy/news/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [  # pylint: disable=invalid-name
```

Comment:
Please remove the comment as this project doesn't need to be consistent with `Pylint`. It is there just for pointing out and we can ignore the things that we wish to.